src/androidTest/java/com/github/aloomaio/androidsdk/aloomametrics/DecideCheckerTest.java

```diff
@@ -322,7 +323,7 @@ private void assertUpdatesSeen(JSONArray[] expected) {
         }
     }
 
-    private static class MockPoster extends ServerMessage {
+    private static class MockPoster extends HttpService {
```

Comment:
Tests can not run due to an error on `TestUtils`. 
Please update line 24 with our new constructor `super(context, referrerPreferences, token, "", false, null);`